tools/RAiDER/aria/calcGUNW.py

```diff
@@ -1,7 +1,69 @@
-def main(args):
-    """ Calculate  interferometric phase delay at dates in ARIA standard product (GUNW) """
-    pass
+"""
+Calculate the interferometric phase from the 4 delays files of a GUNW
+Write it to disk
+"""
+import os
+import xarray as xr
+import numpy as np
+from RAiDER.utilFcns import rio_open
+from RAiDER.logger import logger
 
+# capture method (proj slant zenith) in metadata
+# capture model
+# capture description of variable
+# capture units # radians
+# short name, long name
+# projection
+
+
+def main(dct_delays:dict, path_gunw:str, wavelength, out_dir:str, update_flag):
```

Comment:
I think for now filenames are better. That way a user can break it apart more easily by creating there a simple dictionary of dates to files instead of having to run the whole calculation again to get the dataset

wavelength I don't have a preference